sdv/constraints/errors.py

```diff
@@ -8,7 +8,7 @@ def __init__(self, missing_columns):
         self.missing_columns = missing_columns
 
 
-class MultipleConstraintsErrors(Exception):
+class MultipleConstraintsError(Exception):
```

Comment:
This name kind of makes it sound like the problem is that there are multiple constraints. Could we call this `AggregateConstraintsError` or something like that